OADP - adding a JTBD to Installing OADP (#99440)
openshift/openshift-docs: backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc

[role="_abstract"]

> As a cluster administrator, your primary goal is to ensure the complete recoverability of your stateful applications in the event of a disaster. {oadp-first} is the tool you use to accomplish this job, enabling you to safeguard applications, persistent volumes, and critical Kubernetes resources. This guide focuses on the installation and configuration of {oadp-short}, providing you with a clear, step-by-step path to achieving application resilience and maintaining business continuity with confidence.

Author: ```suggestion
As a cluster administrator, your primary goal is to ensure the complete recoverability of your stateful applications if there is a disaster. {oadp-first} is the tool you use to accomplish this job, enabling you to safeguard applications, persistent volumes, and critical Kubernetes resources. This guide focuses on the installation and configuration of {oadp-short}, providing you with a clear, step-by-step path to achieving application resilience and maintaining business continuity with confidence.
```
